experimental/dds/tree2/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -168,6 +171,23 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends
 			undoRedoManager.clone(() => this.editManager.getTrunkHead()),
 			anchors,
 		);
+		this.editManager.on("newTrunkHead", (head) => {
+			this.changeEvents.emit("newSequencedChange", head.change);
+		});
+		this.editManager.localBranch.on("change", (event) => {
```

Comment:
Can you add a comment for this? All of the events are starting to get really confusing for me.

Reply:
Of course!